wasp/apps/connect4.py

```diff
@@ -0,0 +1,173 @@
+"""Connect 4
+   ~~~~~~~~~~~~~
+    
+    Connect 4 is a 2 player game played on a 6x7 (height x width) grid.
+    The players can choose in which column to drop a disc with their color,
+    and when one of the players gets 4 adjacent discs in a row, column or diagonal, they win.
+    
+    Connect 4 is a solved game, meaning that in every case,
+    by making the right move, the first player can win."""
```

Comment:
Please no indentation here please.

Also please add a screenshot ('s' in the simulator) to the header comment (see any of the apps in the [Application Library](https://wasp-os.readthedocs.io/en/latest/apps.html) for examples of how to mark this up.

Reply:
done